charts/builder/templates/builder-deployment.yaml

```diff
@@ -94,6 +94,10 @@ spec:
                 secretKeyRef:
                   name: builder-key-auth
                   key: builder-key
+{{- if (.Values.builder_pod_node_selector) }}
```

Comment:
not really a big reason for that, it would just be less variable in yml file. but can do either ways.